include/onnxruntime/core/common/const_pointer_container.h

```diff
@@ -64,6 +64,10 @@ class ConstPointerContainer {
   explicit ConstPointerContainer(const Container& data) noexcept : data_(data) {}
 
   size_t size() const noexcept { return data_.size(); }
+  bool empty() const noexcept { return data_.empty(); }
+
+  ConstIterator cbegin() const noexcept { return ConstIterator(data_.cbegin()); }
```

Comment:
are these two needed?

Reply:
> are these two needed?

They make it more consistent with the std Container interfaces. See the Member Function Table here: https://en.cppreference.com/w/cpp/container